src/Type/Accessory/AccessoryNumericStringType.php

```diff
@@ -275,6 +276,10 @@ public function isScalar(): TrinaryLogic
 
 	public function looseCompare(Type $type, PhpVersion $phpVersion): BooleanType
 	{
+		if ($type->isString()->yes() && $type->isNumericString()->no()) {
+			return new ConstantBooleanType(false);
```

Comment:
This one depends on the PhpVersion, right? Because of https://wiki.php.net/rfc/string_to_number_comparison

Reply:
I don't think so, since we compare string with string here.